src/service/translate.js

```diff
@@ -2291,6 +2291,27 @@ function $translate($STORAGE_KEY, $windowProvider, $translateSanitizationProvide
         return null;
       };
 
+      /**
+       * @ngdoc function
+       * @name pascalprecht.translate.$translate#getTranslationTable
+       * @methodOf pascalprecht.translate.$translate
+       *
+       * @description
+       * Returns translation table by a key of the locale
+       * if key is not provided it returns translation table which currently
+       * in use
+       *
+       * @param {string} translationId A token which represents a translation id
+       *
+       * @return {object} angular-translate $translationTable
+       */
+      $translate.getTranslationTable = function (translationId) {
+        if(!translationId) {
+          return $translationTable[$translate.use()];
+        }
+        return $translationTable[translationId];
```

Comment:
Perhaps we should also add a test + doc about the situation when translation are loaded async but `$translate.getTranslationTable(lang)` will return null (i.e. because of loading in progress).